File: solutions/3300-3399/3392_count-subarrays-of-length-three-with-a-condition/solution.go
package leeycode

func countSubarrays(nums []int) int {
	var res int
	for i := 0; i < len(nums)-2; i++ {
		if nums[i+1]%2 == 0 && nums[i]+nums[i+2] == nums[i+1]/2 {
			res++
		}
	}
	return res
}

File: solutions/3300-3399/3392_count-subarrays-of-length-three-with-a-condition/solution.py
class Solution(object):
    def countSubarrays(self, nums):
        res = 0
        for i in range(len(nums)-2):
            if nums[i+1]%2==0 and nums[i]+nums[i+2]==nums[i+1]//2:
                res+=1
        return res